kpi/schema_extensions/v2/members/extensions.py

```diff
@@ -1,12 +1,16 @@
 from drf_spectacular.extensions import OpenApiSerializerFieldExtension
-from drf_spectacular.plumbing import build_basic_type, build_object_type
+from drf_spectacular.plumbing import build_basic_type, build_object_type, \
+    build_choice_field
 from drf_spectacular.types import OpenApiTypes
 
+from kpi.models.import_export_task import ImportExportStatusChoices
 from kpi.schema_extensions.v2.generic.schema import (
     GENERIC_STRING_SCHEMA,
     USER_URL_SCHEMA,
 )
+from rest_framework import serializers
 from kpi.utils.schema_extensions.url_builder import build_url_type
+from .schema import CHOICES, PAYLOAD_CHOICES
```

Comment:
`CHOICES` and `PAYLOAD_CHOICES` are too generic. Please rename them to something more obvious.